icu4j/main/classes/core/src/com/ibm/icu/util/Currency.java

```diff
@@ -90,14 +90,38 @@
 
     /**
      * Selector for getName() indicating the narrow currency symbol.
-     * The narrow currency symbol is similar to the regular currency
-     * symbol, but it always takes the shortest form: for example,
-     * "$" instead of "US$" for USD in en-CA.
+     *
+     * The narrow currency symbol is similar to the regular currency symbol,
+     * but it always takes the shortest form;
+     * for example, "$" instead of "US$" for USD in en-CA.
      *
      * @stable ICU 61
      */
     public static final int NARROW_SYMBOL_NAME = 3;
 
+    /**
+     * Selector for getName() indicating the formal currency symbol.
+     *
+     * The formal currency symbol is similar to the regular currency symbol,
+     * but it always takes the form used in formal settings such as banking;
+     * for example, "NT$" instead of "$" for TWD in zh-TW.
+     *
+     * @provisional This API might change or be removed in a future release.
```

Comment:
FYI optional I guess -- we normally put draft first, then provisional. Tools probably don't care?